dash/development/base_component.py

```diff
@@ -51,30 +51,30 @@ def is_number(s):
 
 
 def _check_if_has_indexable_children(item):
-    if (not hasattr(item, 'children') or
-            (not isinstance(item.children, Component) and
-             not isinstance(item.children, (tuple, MutableSequence)))):
+    if not hasattr(item, "children") or (
+        not isinstance(item.children, Component)
+        and not isinstance(item.children, (tuple, MutableSequence))
+    ):
```

Comment:
```
if (
    not hasattr(item, "children") or
    not isinstance(item.children, (Component, tuple, MutableSequence))
):
```

or even
```
if not isinstance(
    getattr(item, "children", None),
    (Component, tuple, MutableSequence)
):
```
?

Reply:
Alright, I see, this whole diff is just from black... no need to clean it up now but this code definitely needs some attention at some point.